index.html

```diff
@@ -1392,6 +1392,16 @@ <h2>
 <code>publicKeyBase58</code> property.
             </td>
           </tr>
+          <tr>
+            <td>
+Curve25519
+            </td>
+            <td>
+Curve25519 public key values MUST either be encoded as a JWK or be encoded as
+the raw 32-byte public key value in Base58 Bitcoin format using the
+<code>publicKeyBase58</code> property if they are not encoded in JWK format.
```

Comment:
```suggestion
<code>publicKeyBase58</code> property.
```
The trailing phrase is redundant.